Quantum-Control-Applications/Superconducting/Single Fixed Transmon/rb.py

```diff
@@ -141,20 +144,18 @@ def play_sequence(sequence_list, depth):
                     None,
                     dual_demod.full("cos", "out1", "sin", "out2", I),
                 )
-                assign(res, I > 0.1)  # Need to set threshold
-                save(res, res_st)
-
-                assign(sequence_list[depth], saved_gate)
+                state = readout_macro(threshold=ge_threshold, state=state)
+                save(state, state_st)
```

Comment:
I think we need to reset the qubit here? either by waiting or active reset

Reply:
In line 13 there is a wait that does it ...

Reply:
Sorry, that part was hidden for me (because you made no changes there).
But now I see more things:
1. You added the readout_macro, this means you need to remove the measure command...
2. Why did you remove the `assign(sequence_list[depth], saved_gate)`? It is important
3. The `wait(5000, "qubit")` should be replaced by `wait(cooldown_time, "resonator")` and we should have `cooldown_time = 5 * qubit_T1 // 4` (see T1 script for example)